certificate-service/clinicalprocess-certificate-v4/src/test/resources/igrav.v1.sch

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The test classes should not use its own schematron. It should use the actual schematrons for the certificates. Otherwise we might have a schematron that works in the test class but not in production.